charts/kube-master/charts/etcd/templates/deployment.yaml

```diff
@@ -52,7 +52,8 @@ data:
       --listen-client-urls=https://0.0.0.0:2379
 {{- else }}
       --advertise-client-urls=http://${ETCD_IP}:2379 \
-      --listen-client-urls=http://0.0.0.0:2379
+      --listen-client-urls=http://0.0.0.0:2379 \
+      --listen-metrics-urls=http://0.0.0.0:8081
```

Comment:
hmm we have secure enabled by default now. so I think if we leave it like this we don't get metrics for etcd, no? I think we should add the metrics flag in all cases (outside of the if statement). What do you think?

Reply:
you are right, I have missed the `.Values.secure.enabled` bit